src/index.js

```diff
@@ -64,6 +66,15 @@ async function bundleMDX(
           if (fullModulePath in absoluteFiles) return {path: fullModulePath}
         }
 
+        if(cwd){
```

Comment:
I forgot we disabled the git hook. If you want to, you could you run `npx kcd-scripts format` to format everything. Eventually we can add back the git hook.